cypress/support/commands.js

```diff
@@ -10,11 +10,11 @@
 //
 //
 // -- This is a parent command --
-Cypress.Commands.add('login', (email, password) => { 
-    cy.get('[id="Login"]').type(email), 
-    cy.get('[id="Password"]').type(password)
-    cy.get('[value="Zaloguj"]').click()
- })
+// Cypress.Commands.add('login', (email, password) => { 
```

Comment:
Dlaczego to jest zakomentowane?

Reply:
Bo mam te sama funkcje napisana w loginPage więc nie jest mi potrzebna, stąd ją usunę z commands,js